{"name": "test_simple_assume12", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\ntest_xdist.py:61: AssumptionFailure\n>>\tpytest.assume(1==0)\nAssertionError: assert False\n\ntest_xdist.py:63: AssumptionFailure\n>>\tpytest.assume(False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               test_xdist.py:61: AssumptionFailure\nE               >>\tpytest.assume(1==0)\nE               AssertionError: assert False\nE               \nE               test_xdist.py:63: AssumptionFailure\nE               >>\tpytest.assume(False)\nE               AssertionError: assert False\n\nD:\\Miniconda\\Lib\\site-packages\\six.py:718: FailedAssumption"}, "start": 1701430294844, "stop": 1701430294872, "uuid": "0a9452b2-f165-48b2-a7fc-80a72471c660", "historyId": "d893cf9259cd631c83edda663f1aa90e", "testCaseId": "d893cf9259cd631c83edda663f1aa90e", "fullName": "test_xdist#test_simple_assume12", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}